runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -2806,7 +2806,11 @@ class INTERPRETER_CLASS
 		J9Class *receiverClazz = J9VM_J9CLASS_FROM_HEAPCLASS(_currentThread, *(j9object_t*)_sp);
 		j9object_t simpleName = NULL;
 		J9ROMClass *romClass = receiverClazz->romClass;
-		J9UTF8 *simpleNameUTF = getSimpleNameForROMClass(_vm, receiverClazz->classLoader, romClass);
+		J9UTF8 *simpleNameUTF = NULL;
+		if (!J9ROMCLASS_IS_HIDDEN(romClass)) {
+			/* hidden simple name of a hidden class */
+			simpleNameUTF = getSimpleNameForROMClass(_vm, receiverClazz->classLoader, romClass);
+		}
 		if (NULL != simpleNameUTF) {
 			buildInternalNativeStackFrame(REGISTER_ARGS);
 			updateVMStruct(REGISTER_ARGS);
```

Comment:
Is this meant to always return null for the hidden classes simple name?
```suggestion
		
		if (!J9ROMCLASS_IS_HIDDEN(romClass)) {
			J9UTF8 *simpleNameUTF = getSimpleNameForROMClass(_vm, receiverClazz->classLoader, romClass);
			if (NULL != simpleNameUTF) {
				buildInternalNativeStackFrame(REGISTER_ARGS);
				updateVMStruct(REGISTER_ARGS);
				.....
			}
		}
```

Reply:
Yes, always return null for the hidden classes simple name. Fixed as suggested.